Attendance.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+
+const Attendance = () => {
+  const [student, setStudent] = useState('');
+  const [status, setStatus] = useState('');
+
+  const markAttendance = async (e) => {
+    e.preventDefault();
+
+    const endpoint = status === 'present' ? '/api/attend' : '/api/absent';
+    await axios.post(endpoint, { student });
```

Comment:
Consider adding error handling for the axios post request. In case the request fails, the error should be caught and handled appropriately.